### PR TITLE
Add tokenURI() method and tests for MyCollectible.

### PR DESCRIPTION
**Pull request by robmyers**
Does what it says. :-)

### CODE REVIEW

File: contracts/MyCollectible.sol [hunk 1]
```diff
@@ -7,11 +7,16 @@ import "./ERC1155Tradable.sol";
...
+  ERC1155Tradable(
     "MyCollectible",
     "MCB",
     _proxyRegistryAddress
   ) public {
     _setBaseMetadataURI("https://opensea-creatures-api.herokuapp.com/api/creature/");
   }
+
+  function contractURI() public view returns (string memory) {
+    return "https://opensea-creatures-api.herokuapp.com/contract/opensea-erc1155";
```
Comment (dfinzer) [inline contracts/MyCollectible.sol:20]: Let's use `creatures-api.opensea.io`

File: test/MyCollectible.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/* Contracts in this test */
+
+const MyCollectible = artifacts.require("../contracts/MyCollectible.sol");
+
+
+contract("MyCollectible", (accounts) => {
+  const URI_BASE = 'https://opensea-creatures-api.herokuapp.com';
```
Comment (dfinzer) [inline test/MyCollectible.js:7]: New URI :)

File: contracts/MyCollectible.sol (hunk 1 above)
Comment (robmyers) [reply contracts/MyCollectible.sol]: Fixed.

File: test/MyCollectible.js (hunk 2 above)
Comment (robmyers) [reply test/MyCollectible.js]: Fixed.

**Pull request merged**